package.json

```diff
@@ -30,35 +30,40 @@
     "test:debug": "node --inspect-brk -r tsconfig-paths/register -r ts-node/register node_modules/.bin/jest --runInBand"
   },
   "dependencies": {
-    "@nestjs/common": "^9.3.12",
-    "@nestjs/core": "^9.3.12",
-    "ottoman": "^2.3.2",
-    "reflect-metadata": "^0.1.13",
-    "rxjs": "^7.8.0"
+    "reflect-metadata": "^0.2.2",
+    "rxjs": "^7.8.1"
+  },
+  "peerDependencies": {
```

Comment:
👍 
can you help to add `engines.code` to make sure node version >=18.

I recall npm behaviour about peerDependencies changed around npm v7(probably node v16~ish).

Reply:
✅️